src/Symfony/Component/Console/Application.php

```diff
@@ -596,6 +596,19 @@ public function find($name)
         $this->init();
 
         $aliases = [];
+
+        foreach ($this->commands as $command) {
+            foreach ($command->getAliases() as $alias) {
+                if (!$this->has($alias)) {
+                    $this->commands[$alias] = $command;
+                }
+            }
+        }
+
+        if ($this->has($name)) {
+            return $this->get($name);
+        }
```

Comment:
This change is about optimization only, should be done on master instead (removing the `$exact` logic below as it becomes unreachable)